Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN mkdir -p /app
 WORKDIR /app
 ADD . .
 
-RUN npm ci --no-audit && \
+RUN CYPRESS_INSTALL_BINARY=0 npm ci --no-audit && \
```

Comment:
@andrewrlee better way of not installing cypress rather than excluding from package.json